### DISCUSSION TITLE
Should I use null and not undefined for nullable properties?

### DISCUSSION THREAD

**Discussion by vatosarmat** [vuejs/pinia#1931] [Help and Questions] [answered]
I've just started reading the documentation and wondering why it uses `null` instead of `undefined` in examples for nullable properties? Is this just a convention or requirement for reactivity to work?

**Comment by posva** [comment, marked as answer]
It's just a convention and, IMO, a good practice:

- `null`: intentionally doesn't exist/empty
- `undefined`: still hasn't been given a value
